keycloak.yaml

```diff
@@ -62,6 +62,147 @@ subpackages:
       - runs: |
           mkdir -p "${{targets.subpkgdir}}"/opt
           ln -s /usr/share/java/keycloak "${{targets.subpkgdir}}"/opt/
+  # https://github.com/bitnami/containers/tree/main/bitnami/keycloak/24/debian-12
+  - name: ${{package.name}}-bitnami-compat
+    description: "compat package with bitnami/keycloak image"
+    dependencies:
+      runtime:
+        - krb5
+        - libaio
+        - procps
+        - zlib
+        - wait-for-port
+        - net-tools
+        - posix-libc-utils
+        - su-exec
```

Comment:
keycloak-compat shoudl depend on keycloak?

Reply:
No actually, for the version-streaming, it should be added in the image manifest

Reply:
I admit to not fully understanding version-streaming. but why would you not add the dependency here? you can't use keycloak-bitnami-compat without keycloak.

Reply:
> you can't use keycloak-bitnami-compat without keycloak.

Correct, so we need to install both of at the same time.

> but why would you not add the dependency here?

If we have that dep here, we would copy the entire bitnami-compat for other versions by including version suffix, `keycloak-bitnami-compat-X.Y`. We also wouldn't consume this subpkg for FIPS package. By not including the `keycloak` package in the runtime deps, we would only have 1 compat package, without needing duplicating.